src/types/Props.ts

```diff
@@ -0,0 +1,38 @@
+import { Todo } from './Todo';
+import { FilterStates } from '../types/enums';
+
+export interface ErrorNotificationProps {
+  error: string | null;
+  onHideError: () => void;
+}
+
+export interface FooterProps {
+  todos: Todo[];
+  filter: FilterStates;
+  setFilter: React.Dispatch<React.SetStateAction<FilterStates>>;
```

Comment:
use this syntax for all such cases. More readable
```suggestion
  setFilter: (filterStates: FilterStates) => void;
```